diff -Naur guess_fs-20140626/dosfsck.h guess_fs-20140626-patched/dosfsck.h
--- guess_fs-20140626/dosfsck.h	2014-06-25 09:55:21.000000000 +0000
+++ guess_fs-20140626-patched/dosfsck.h	2016-05-13 18:20:30.105579856 +0000
@@ -10,9 +10,7 @@
 #define _DOSFSCK_H
 
 #include <sys/types.h>
-#define _LINUX_STAT_H		/* hack to avoid inclusion of <linux/stat.h> */
-#define _LINUX_STRING_H_	/* hack to avoid inclusion of <linux/string.h>*/
-//#include <linux/msdos_fs.h>
+#include <linux/msdos_fs.h>
 
 /* 2.1 kernels use le16_to_cpu() type functions for CF_LE_W & Co., but don't
  * export this macros, only __le16_to_cpu(). */
diff -Naur guess_fs-20140626/linux_fs.h guess_fs-20140626-patched/linux_fs.h
--- guess_fs-20140626/linux_fs.h	2014-06-25 09:55:21.000000000 +0000
+++ guess_fs-20140626-patched/linux_fs.h	2016-05-13 18:00:40.385617451 +0000
@@ -3,6 +3,8 @@
    only designed to be able to check a magic number
    in case no filesystem type was given. */
 
+#include <stdint.h>
+
 typedef unsigned char u_char; /*BK*/
 
 #ifndef BLKGETSIZE
diff -Naur guess_fs-20140626/Makefile guess_fs-20140626-patched/Makefile
--- guess_fs-20140626/Makefile	2014-06-26 02:07:29.000000000 +0000
+++ guess_fs-20140626-patched/Makefile	2016-05-13 18:00:14.565616144 +0000
@@ -1,13 +1,12 @@
-CC = musl-gcc
-#CC = musl-gcc --static
+CC = gcc
 CFLAGS = -W -Wall -Os -D_GNU_SOURCE -fno-strict-aliasing -Wno-multichar
+LDFLAGS =
 EXE = guess_fstype
 
 default: $(EXE) tidy
 
 $(EXE): guess_fstype.c main.c Makefile
-	@echo "Single compile for guess_fstype ..."
-	@$(CC) -o $@ guess_fstype.c main.c $(CFLAGS) -s
+	@$(CC) -o $@ guess_fstype.c main.c $(CFLAGS) $(LDFLAGS) -s
 	@ls -l $@
 
 tidy:
